sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -376,102 +374,115 @@ The `node` class provides the {crs}[common reference semantics].
 
 ==== Node Member Functions
 
-Table {counter: tableNumber}. Member functions of the `node` class.
-[cols="2a,a"]
-|===
-|Member Function|Description
-
-|
-[source,c++]
+|====
```

Comment:
I don't find the rendering of the tables very readable now with a small code snippet in a larger cell of a single column table
![image](https://github.com/user-attachments/assets/a6d7c995-c998-4850-9dfe-7c62782a4df3)

If we're only using a single column of a table, a table the best format for showing things? I'm not seeing tables used in other specs with the new format.

Reply:
I see both actually used for recent extensions, with and without tables being used.

No strong opinion here, let's go with the simple approach and don't use a table.